project_name/settings.py

```diff
@@ -11,20 +11,30 @@
 """
 
 import os
+from os.path import dirname, join, exists, abspath
+
+import environ  
+
+# Load operating system env variables and prepare to use them
+env = environ.Env()
+
+# .env file, should load only in development environment
+env_file = join(dirname(__file__), 'local.env')
```

Comment:
Well, I believe production servers should never rely on .env files, only on actual environment variables, so I use `local.env` to hammer home the idea.  But I'm totally cool with just `.env` too.